packages/block-library/src/cover/editor.scss

```diff
@@ -12,6 +12,21 @@
 		color: inherit;
 	}
 
+	// Applied while media is being uploaded
+	&.is-transient::before {
+		background-color: #fff;
+		opacity: 0.3;
+	}
+
+	// Shown while media is being uploaded
+	.components-spinner {
+		position: absolute;
+		top: 50%;
+		left: 50%;
+		margin-top: -9px;
+		margin-left: -9px;
```

Comment:
I have updated the code(423592c) to:
1. Make the positioning more abstract (independent of spinner width / height)
2. Document the the need of `transform: translate(-50%, -50%);` for (1)

Reply:
That's a better way to handle it 👍

Reply:
Is this because the spinner is 18px wide, and 9px is half of that? We have `$spinner-size`, and it'd be nice then if you used `$spinner-size / 2` instead of 9px.

Reply:
> Is this because the spinner is 18px wide, and 9px is half of that?

@jasmussen exactly. I eventually used `transform: translate(-50%, -50%);` which is independent of the spinner-size.